spa/src/components/common/IntegrationCard/MailchimpIntegrationCard/MailchimpIntegrationCard.tsx

```diff
@@ -13,7 +14,7 @@ export function MailchimpIntegrationCard() {
       image={MailchimpLogo}
       title="Mailchimp"
       isRequired={false}
-      cornerMessage={freePlan ? 'Upgrade to Core' : undefined}
+      cornerMessage={!isLoading && freePlan ? <CornerMessage>Upgrade to Core</CornerMessage> : undefined}
```

Comment:
Think we could simplify this a bit
```suggestion
      cornerMessage={!isLoading && freePlan && <CornerMessage>Upgrade to Core</CornerMessage>}
```

Reply:
Done in 88742d2d5a8d5d9234ecb8afcfc980ff451340c2